lib/vzaar_api/category.rb

```diff
@@ -9,17 +9,30 @@ class Category < Abstract
                     :node_video_count, :tree_video_count,
                     :created_at, :updated_at].freeze
 
+    ATTR_ACCESSORS = [:name, :parent_id, :move_to_root].freeze
+
     prepend Lib::HasAttributes
     include Lib::HasResourceUrl
     include Lib::ActiveObject::Find
+    include Lib::ActiveObject::Save
+    include Lib::ActiveObject::Delete
     include Lib::WillPaginate
 
+    def initialize(attrs = {})
+      update_from_attributes attrs
+    end
+
+    def self.create(attrs = {})
+      Strategy::Category::Create.new(attrs, self).execute
+    end
+
     def subtree(query = {})
       args = query.merge({
         resource_url: resource_url("#{id}/subtree"),
         resource_class: self.class })
       Lib::PagedResource.new(args)
     end
 
+    def update_from_attributes(attrs); end
```

Comment:
Yeah, we get an error when the `HasAttributes` module tries to call the superclass on update. 

The other option is to put this into `VzaarApi::Abstract` along with the empty init method.